internal/commands/scan_test.go

```diff
@@ -689,7 +689,7 @@ func TestAddSCSScan_ResubmitWithOutScorecardFlags_ShouldPass(t *testing.T) {
 		},
 	}
 
-	result, _ := addSCSScan(cmdCommand, resubmitConfig)
+	result, _ := addSCSScan(cmdCommand, resubmitConfig, true)
```

Comment:
please also add unit tests that check the logic when enterprise license is false

Reply:
done